File: Client/Assets/Resources/Script/WebAPI/DTO/Rank/RankedRecordDto.cs
using System;
using System.Collections.Generic;

public class RankedRecordDto
{
    public int recordId {get; set;}
    public int recordRank {get; set;}
    public int recordTime { get; set;}
    public string recordHistory { get; set;}
    public List<RecordUserDto> users { get; set;}
}

File: Server/webserver/src/main/java/com/onlyone/gameserver/api/dto/record/RankedRecordDto.java
package com.onlyone.gameserver.api.dto.record;

import com.onlyone.gameserver.api.dto.user.RecordUserDto;
import lombok.AllArgsConstructor;
import lombok.Data;

import java.time.LocalDateTime;
import java.util.List;

@Data
@AllArgsConstructor
public class RankedRecordDto {
    private Integer recordId;
    private Integer recordRank;
    private Integer recordTime;
    private LocalDateTime recordHistory;
    private List<RecordUserDto> users;

}
